frontend/src/components/EventForm/EventForm.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import { Box, Typography } from '@mui/material';
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+import createNewEvent from '../../requests/post/createNewEvent';
+import Alert from '@mui/material/Alert';
+import './style.css';
+
+function EventForm() {
+    const [title, setTitle] = useState('');
+    const [description, setDescription] = useState('');
+    const [date, setDate] = useState("");
+    const [time, setTime] = useState("");
+    const [image, setImage] = useState("");
+
+    const [successAlert, setSuccessAlert] = useState(false);
+    const [errorAlert, setErrorAlert] = useState(null);
+
+    const handleSubmit = () => {
+        console.log(title, description, date, time, image)
+        createNewEvent(title, description, date, time, image)
+        .then(res => {
+            if(res == 'OK') {
+                setSuccessAlert('Event created successfuly!');
+                setErrorAlert(null);
+            }
+        })
+        .catch(err => {
+            setSuccessAlert(false);
+            setErrorAlert(err.message);
+        });
+    };
+
+  return (
+    <>
+        <Box className="event-form-container">
+            <Typography variant="h3" sx={{ mt: 5 }}>
+                Create New Event
+            </Typography>
+            <TextField
+                id="standard-basic"
+                label="Title"
+                className="event-form-input"
+                sx={{ mt: 3 }}
+                variant="standard"
+                value={title}
+                onChange={(e) => setTitle(e.target.value)}
+            />
+            <br />
+            <TextField
+                id="standard-basic"
+                label="Description"
+                className="event-form-input"
+                sx={{ mt: 3 }}
+                variant="standard"
+                value={description}
+                onChange={(e) => setDescription(e.target.value)}
+            />
+            <br />
+            <label style={{ marginTop: '2rem' }}>Date: </label>
+            <input
```

Comment:
Past dates should be unavailable

Reply:
Inside backend/app/Http/Requests/StoreEventRequest.php and UpdateEventRequest.php I added rule that date has to be **after_or_equal:today**